tests/tests-core/src/main/java/com/gentics/mesh/core/node/NodeImageResizeEndpointTest.java

```diff
@@ -232,7 +232,7 @@ public void testTransformImage() throws Exception {
 		ImageManipulationParameters params = new ImageManipulationParametersImpl().setWidth(100);
 		NodeResponse transformResponse = call(() -> client().transformNodeBinaryField(PROJECT_NAME, nodeUuid, "en", version, "image", params));
 		assertEquals("The image should have been resized", 100, transformResponse.getFields().getBinaryField("image").getWidth().intValue());
-		MeshCoreAssertion.assertThat(testContext).hasUploads(2, 2).hasTempFiles(0).hasTempUploads(0);
+		MeshCoreAssertion.assertThat(testContext).hasUploads(1, 1).hasTempFiles(0).hasTempUploads(0);
```

Comment:
Why did this change?

Reply:
As we now process the binary files correctly, the old unused binary is no more there.